src/MCRun.jl

```diff
@@ -94,7 +115,10 @@ end
 
 """
     function mc_step!(mc_states,mc_params,pot,ensemble)
+        (nnp_states::NNPState,move_strat,mc_params,potential,ensemble)
```

Comment:
again repeat function name, also above -  it is just easier to read, and I think it's also the convention